src/lib/Microsoft.Fx.Portability/Proxy/ProxyAuthenticationHandler.cs

```diff
@@ -41,7 +41,12 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
             {
                 try
                 {
-                    return await base.SendAsync(request, cancellationToken);
+                    HttpResponseMessage result = await base.SendAsync(request, cancellationToken);
+                    if (result.StatusCode == HttpStatusCode.ProxyAuthenticationRequired)
+                    {
+                        throw new ProxyAuthenticationRequiredException(request.RequestUri);
+                    }
```

Comment:
Style Cop seems to want a blank line after this (which seems a little silly)

```suggestion
                    }

```